Documentation/Index.rst

```diff
@@ -67,6 +67,32 @@ For compatibility reasons, this extension is installable in TYPO3 v11 already.
 If installed in TYPO3 v12, additional CSS is loaded that provides the
 :css:`fa-*` classes.
 
+Usage
+=====
+
+To register icons from FontAwesome create a file called
+:file:`Configuration/Icons.php` in your extension (if it does not exist yet):
+
+.. code-block:: php
+   :caption: EXT:your_extension/Configuration/Icons.php
+
+   <?php
+   return [
+       // Icon identifier
+       'myfontawesomeicon' => [
+           'provider' => \FriendsOfTYPO3\FontawesomeProvider\Imaging\IconProvider\FontawesomeIconProvider::class,
+           // The FontAwesome icon name
+           'name' => 'spinner',
+           // Additional css classes
+           'additionalClasses' => 'fa-fw',
```

Comment:
Do you have a hint on where this is used? I cannot find this in the icon provider, nor in Core. I see this is documented at https://docs.typo3.org/m/typo3/reference-coreapi/11.5/en-us/ApiOverview/Icon/Index.html which looks wrong to me.

Reply:
I just migrated the usage from the mentioned documentation. But you are right, "additionalClasses" is not available in the code. I will provide an update of the patch and take care of the "TYPO3 Explained" documentation.